pyblish_deadline/plugins/deadline.py

```diff
@@ -92,7 +92,7 @@ def process(self, context):
             if instance.has_data('deadlineFrames'):
                 job_data['Frames'] = instance.data('deadlineFrames')
 
-            if not job_data['Plugin']:
+            if not job_data.get('Plugin'):
```

Comment:
I was always under the impression that get method on dictionary automatically returns None if not found and you use 'default' flag only if you want to return a different value.

Quite frankly I don't mind either of the solution that you suggested. As far as they all work (they should), I think it's splitting hairs.